### ISSUE TITLE
[BUG]: Mod Profile Manager - Crash shortly after loading mods, on main menu.

### ISSUE THREAD

**Issue by algramic** [Sgt-Imalas/Sgt_Imalas-Oni-Mods#88]
### Which Mod?

Mod Profile Manager

### Description of the bug/crash

Happens on the latest version of the mod, 6.0. I downloaded the old version, 5.1, and it loaded just fine.

```
NullReferenceException: Object reference not set to an instance of an object

SaveGameModLoader.AllPatches+Steam_MakeMod.Postfix (KMod.Mod __result) (at <2578084d3a7e454fbe440d61925716b0>:0)
(wrapper dynamic-method) KMod.Steam.KMod.Steam.MakeMod_Patch1(KMod.Steam,SteamUGCService/Mod)
(wrapper dynamic-method) KMod.Steam.KMod.Steam.UpdateMods_Patch0(KMod.Steam,System.Collections.Generic.IEnumerable`1<Steamworks.PublishedFileId_t>,System.Collections.Generic.IEnumerable`1<Steamworks.PublishedFileId_t>,System.Collections.Generic.IEnumerable`1<Steamworks.PublishedFileId_t>,System.Collections.Generic.IEnumerable`1<SteamUGCService/Mod>)
(wrapper dynamic-method) SteamUGCService.SteamUGCService.Update_Patch1(SteamUGCService)

Build: U50-583005-S
```

### Steps To Reproduce

Open game
Wait for main menu to load
Wait a few seconds
Crash.

### Player.log

https://gist.github.com/algramic/5e405fa57c22ab515c782add43c73929

**Comment by Sgt-Imalas** [comment]
I have released update 6.1 which potentially fixes this crash. 
could you test that new version=

**Comment by algramic** [comment]
Yes, the updated version fixed the crash. Thank you :)
